frontend/assets/javascripts/src/modules/form/validation/existEmail.es6

```diff
@@ -0,0 +1,32 @@
+import ajax from 'ajax';
+import debounce from 'lodash/debounce'
+
+export function init() {
+    const emailField = document.getElementById('email');
+    const isUseErrorMessage = document.getElementById('form-field__error-message-email-checker');
+    const checkerURL = '/user/check-existing-email';
+
+    const validateEmail = debounce((event) => {
+        debugger;
+        let prospectiveEmail = emailField.value;
+
+        if(!prospectiveEmail.includes('@') || !prospectiveEmail.includes('.')) {
+            return;
+        }
+
+        ajax({
+            url: checkerURL + '?email=' + emailField.value,
```

Comment:
@Ap0c Currently the email checker service is hard-coded and the response does not depend on the input. Once the service is implemented, we will be able to test if we need to URL-enconded or not this parameter. This PR should include that service logic, I will talk with @rtyley about this logic.